packages/catalog-outbound-writer/src/converters/toOutboundEventV2.ts

```diff
@@ -46,97 +46,114 @@ function toOutboundEServiceV2(
-      (msg) => ({
-        event_version: msg.event_version,
-        type: msg.type,
-        version: msg.version,
-        data: {
-          descriptorId: msg.data.descriptorId,
-          documentId: msg.data.documentId,
-          eservice:
-            msg.data.eservice && toOutboundEServiceV2(msg.data.eservice),
-        },
-        stream_id: msg.stream_id,
-        timestamp: new Date(),
-      })
-    )
-    .with(
-      { type: "EServiceRiskAnalysisAdded" },
-      { type: "EServiceRiskAnalysisDeleted" },
-      { type: "EServiceRiskAnalysisUpdated" },
-      () => undefined
-    )
-    .exhaustive();
+      }))
+      .with(
+        { type: "EServiceDescriptorAdded" },
+        { type: "EServiceDraftDescriptorUpdated" },
+        { type: "EServiceDescriptorQuotasUpdated" },
+        { type: "EServiceDescriptorActivated" },
+        { type: "EServiceDescriptorArchived" },
+        { type: "EServiceDescriptorPublished" },
+        { type: "EServiceDescriptorSuspended" },
+        { type: "EServiceDraftDescriptorDeleted" },
+        (msg) => ({
+          event_version: msg.event_version,
+          type: msg.type,
+          version: msg.version,
+          data: {
+            descriptorId: msg.data.descriptorId,
+            eservice:
+              msg.data.eservice && toOutboundEServiceV2(msg.data.eservice),
+          },
+          stream_id: msg.stream_id,
+          timestamp: new Date(),
+        })
+      )
+      .with(
+        { type: "EServiceDescriptorInterfaceAdded" },
+        { type: "EServiceDescriptorDocumentAdded" },
+        { type: "EServiceDescriptorInterfaceUpdated" },
+        { type: "EServiceDescriptorDocumentUpdated" },
+        { type: "EServiceDescriptorInterfaceDeleted" },
+        { type: "EServiceDescriptorDocumentDeleted" },
+        (msg) => ({
+          event_version: msg.event_version,
+          type: msg.type,
+          version: msg.version,
+          data: {
+            descriptorId: msg.data.descriptorId,
+            documentId: msg.data.documentId,
+            eservice:
+              msg.data.eservice && toOutboundEServiceV2(msg.data.eservice),
+          },
+          stream_id: msg.stream_id,
+          timestamp: new Date(),
+        })
+      )
+      // .with({ type: "EServiceDescriptorAttributesUpdated" }, (msg) => ({
+      //   event_version: msg.event_version,
+      //   type: msg.type,
+      //   version: msg.version,
+      //   data: {
+      //     descriptorId: msg.data.descriptorId,
+      //     attributeIds: msg.data.attributeIds,
+      //     eservice: msg.data.eservice && toOutboundEServiceV2(msg.data.eservice),
+      //   },
+      //   stream_id: msg.stream_id,
+      //   timestamp: new Date(),
+      // }))
+      .with(
+        { type: "EServiceRiskAnalysisAdded" },
+        { type: "EServiceRiskAnalysisDeleted" },
+        { type: "EServiceRiskAnalysisUpdated" },
+        { type: "EServiceDescriptorAttributesUpdated" },
```

Comment:
any reason why this is not propagated?

Reply:
I'm waiting for the release of the new version of the outbound models that contains the `EServiceDescriptorAttributesUpdated` event

Reply:
The outbound models PR is merged and the new version is released, I updated the outbound writer